samples/msal-node-samples/nextjs-rsc-ts/src/app/auth/eventConsent/route.ts

```diff
@@ -0,0 +1,10 @@
+import { redirect } from "next/navigation";
+import { NextRequest } from "next/server";
+import { loginRequest } from "~/serverConfig";
+import { authProvider } from "~/services/auth";
+
+export async function GET(request: NextRequest) {
+  const returnTo = new URL("/event", request.url).toString();
+
+  redirect(await authProvider.getAuthCodeUrl(loginRequest, returnTo));
```

Comment:
> redirect(await authProvider.getAuthCodeUrl(loginRequest, returnTo));

Did you mean to use `calendarRequest` here instead?

Reply:
Yes, fixed